README.md

```diff
@@ -34,9 +36,26 @@ for example `~/local/bin`, set the `BINDIR` environment variable like this:
 curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | BINDIR=~/local/bin sh
 ```
 
+#### Using the latest release links
+
+In order to get the latest stable release for your platform use the following links:
+
+- [Linux 64 bit](https://downloads.arduino.cc/arduino-cli/arduino-cli_latest_Linux_64bit.tar.gz)
+- [Linux 32 bit](https://downloads.arduino.cc/arduino-cli/arduino-cli_latest_Linux_32bit.tar.gz)
+- [Linux ARM 64 bit](https://downloads.arduino.cc/arduino-cli/arduino-cli_latest_Linux_ARM64.tar.gz)
+- [Linux ARM 32 bit](https://downloads.arduino.cc/arduino-cli/arduino-cli_latest_Linux_ARMv7.tar.gz)
+- [Windows 64 bit](https://downloads.arduino.cc/arduino-cli/arduino-cli_latest_Windows_64bit.zip)
+- [Windows 32 bit](https://downloads.arduino.cc/arduino-cli/arduino-cli_latest_Windows_32bit.zip)
+- [Mac OSX](https://downloads.arduino.cc/arduino-cli/arduino-cli_latest_macOS_64bit.tar.gz)
+
+These links return a `302: Found` response, redirecting to latest generated builds by replacing `latest` with the latest
```

Comment:
Maybe I'm missing a use case where the information is necessary, but it seems like an unnecessary level of detail about the mechanics behind the URLs. What do you think about simplifying it:
>These links redirect to the download of the latest stable release.

Reply:
If you are using the latest link inside some automation with a raw `curl` to download the package, would be good to know that you will receive a real redirect. This way you can configure your automation to follow it.

I thought that in this use case is better to be super explicit.

I discussed this point also with @masci in this pr Comment https://github.com/arduino/arduino-cli/pull/381#discussion_r319421493. Let me know if this use case makes sense to you.